AU2/plugins/CorePlugin.py

```diff
@@ -243,6 +258,8 @@ def on_assassin_update(self, assassin: Assassin, htmlResponse: Dict) -> List[HTM
         assassin.water_status = htmlResponse[self.html_ids["Water Status"]]
         assassin.college = htmlResponse[self.html_ids["College"]]
         assassin.notes = htmlResponse[self.html_ids["Notes"]]
+        if self.html_ids["Police"] in htmlResponse:
+            assassin.is_police = htmlResponse[self.html_ids["Police"]]

```

Comment:
Force default still populates the field in the response

```suggestion
        assassin.is_police = htmlResponse[self.html_ids["Police"]]
```